src/api/storageMappings.ts

```diff
@@ -21,14 +25,15 @@ const getStorageMappings = (
             catalog_prefix,
             updated_at
         `
-            // TODO (storage mappins) including count will make pagination work but
+            // TODO (storage mappings) including count will make pagination work but
```

Comment:
Did you test if adding the `catalog_prefix` helps at all with this performance? I am assuming not but wanted to check.

Reply:
No, I have not evaluated the performance impact yet.